lib/rust/ensogl/component/grid-view/src/simple.rs

```diff
@@ -171,6 +171,7 @@ impl crate::Entry for Entry {
             data.label.set_content_truncated <+ all(&content, &max_width_px);
 
             out.contour <+ contour;
+            out.highlight_contour <+ contour.map(|c| entry::Contour {size: c.size + Vector2(4.0, 4.0), corners_radius: 2.0});
```

Comment:
These constants should be named and explained.

Reply:
Actually, they shall be removed. I added them only for debugging purpose.